patches/extra/ungoogled-chromium/add-extra-channel-info.patch

```diff
@@ -0,0 +1,30 @@
+description: change extra channel info to "Ungoogled Chromium"
+author: Wengling Chen <feiyu2817@gmail.com>
+
+---
+ chrome/common/channel_info_posix.cc | 2 +-
+ chrome/common/channel_info_win.cc   | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+--- a/chrome/common/channel_info_posix.cc
++++ b/chrome/common/channel_info_posix.cc
+@@ -21,7 +21,7 @@ version_info::Channel GetChannelImpl(std::string* modifier_out) {
+   version_info::Channel channel = version_info::Channel::UNKNOWN;
+   std::string modifier;
+ 
+-  char* env = getenv("CHROME_VERSION_EXTRA");
++  char* env = "Ungoogled Chromium";
+   if (env)
+     modifier = env;
+ 
+--- a/chrome/common/channel_info_win.cc
++++ b/chrome/common/channel_info_win.cc
+@@ -22,7 +22,7 @@ std::string GetChannelName() {
+ #endif  // defined(DCHECK_IS_CONFIGURABLE)
+   return base::UTF16ToASCII(channel);
+ #else
+-  return std::string();
++  return std::string("Ungoogled Chromium");
```

Comment:
We should probably test this on Windows to make sure it won't override the configured channel info.

Reply:
I cannot compile the WIndows version. Who is the maintainer of Win repo? Maybe he can help testing this.

Reply:
I'm maintaining the Windows repo. I can test this for you.